providers/tutorial/units/jobs.pxu

```diff
@@ -0,0 +1,52 @@
+id: tutorial/passing
+plugin: shell
+_description: A job that always passes
+command: true
+
+id: tutorial/failing
+plugin: shell
+_description: A job that always fails
+command: false
+
+id: tutorial/crashing
+plugin: shell
+_description: A job that always crashes
+command:
+ # Start the sleep 5 command, but if it doesn't finish in 1 second, send it the
+ # KILL signal to stop it immediately." Since sleep 5 is meant to run for 5
+ # seconds and the timeout is set to 1 second, the sleep 5 command will be
+ # killed by the KILL signal after just 1 second.
+ timeout -s KILL 0.1 sleep 5
+
+id: tutorial/has_resource
+plugin: shell
+requires:
+ (tutorial_resources.name == 'first-resource' and tutorial_resources.has_required_thing == 'true')
+_description: A job that requires a resource and it is available
+command:
+ true  # this will run!
+
+
+id: tutorial/no_resource
+plugin: shell
+requires:
+ (tutorial_resources.name == 'second-resource' and tutorial_resources.has_required_thing == 'true')
```

Comment:
I was very surprised to see this works. I always assumed Checkbox would check that **any** `tutorial_resources.name` is set to `second-resource` and **any** `tutorial_resources.has_required_thing` is set to `true`, but apparently it's smart enough to understand we want the value of `tutorial_resources.has_required_thing` for the resource named `second-resource`... Interesting!

Reply:
Yep, that's one of the "features" of checkbox. And one that is extremely taxing when you use too many resources :)